uikit/common/sdk/src/main/java/com/sygic/ui/common/sdk/model/ExtendedMapDataModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.sygic.ui.common.sdk.model
+
+import androidx.annotation.RestrictTo
+import androidx.lifecycle.MutableLiveData
+import com.sygic.sdk.map.data.SimpleMapDataModel
+import com.sygic.ui.common.sdk.data.PoiData
+import com.sygic.ui.common.sdk.mapobject.OnClickMapMarker
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class ExtendedMapDataModel : SimpleMapDataModel() {
+
+    private var currentOnClickMapMarker: OnClickMapMarker? = null
+
+    val poiDataObservable: MutableLiveData<PoiData> = MutableLiveData()
```

Comment:
deklarovane by to malo byt iba ako `LiveData` namiesto `MutableLiveData`. takto to dovolis menit kazdemu aj mimo metody `notify`

Reply:
alebo spravit private set